src/io/xpi.js

```diff
@@ -43,53 +43,51 @@ export class Xpi extends IOBase {
     }
     if (this.entries.includes(entry.fileName)) {
       log.info('Found duplicate file entry: "%s" in package', entry.fileName);
-      reject(new Error(oneLine`DuplicateZipEntry: Entry
-        "${entry.fileName}" has already been seen`));
+      throw new Error(oneLine`DuplicateZipEntry: Entry
```

Comment:
besides changing this in a `reject(new Error(...))`, we should probably also return early just after calling `reject(...)`.

Reply:
Done